rllib/agents/mbmpo/tests/test_mbmpo.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+
+import ray
+import ray.rllib.agents.mbmpo as mbmpo
+from ray.rllib.utils.test_utils import check_compute_single_action, \
+    framework_iterator
+
+
+class TestMBMPO(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        ray.init()
+
+    @classmethod
+    def tearDownClass(cls):
+        ray.shutdown()
+
+    def test_mbmpo_compilation(self):
+        """Test whether an MBMPOTrainer can be built with all frameworks."""
+        config = mbmpo.DEFAULT_CONFIG.copy()
+        config["num_workers"] = 2
+        config["horizon"] = 200
+        config["dynamics_model"]["ensemble_size"] = 2
+        num_iterations = 1
+
+        # Test for torch framework (tf not implemented yet).
+        for _ in framework_iterator(config, frameworks="torch"):
+            trainer = mbmpo.MBMPOTrainer(
+                config=config,
+                env="ray.rllib.examples.env.mbmpo_env.HalfCheetahWrapper")
```

Comment:
We will need to change this to Cartpole in the future.

Reply:
+1